tests/Feature/Api/MediaTest.php

```diff
@@ -38,6 +50,37 @@ protected function setUp(): void
         ]);
         $this->file = UploadedFile::fake()->image('TestFile.png');
 
+        $language = Language::factory()->create();
```

Comment:
why do you add this code block to the setUp() method? $this->order is only used once, so this can be done in this corresponding method.